### ISSUE TITLE
Can't find documentation for requests log (devtools.html)

### ISSUE THREAD

**Issue by sander85** [uBlock-LLC/uBlock#1103]
Hi!

For me it's a bit hard to understand how to read the requests log and I can't find any documentation about it.

If the request is allowed by the dynamic filtering then the row's background is green. It seems to be red when the request is blocked by static or dynamic filtering. But what does the gray background mean? Or even better, the white one? I could guess that the white one is noop, but not all rows match that logic either (i.e., I have the default deny rule (\* \* 3p block) and I've enabled behind-the-scene blocking as well, but I can still see some white rows about favicons that shouldn't be allowed). So the log seems confusing.

It would be great to have some legend for those colors.

**Comment by gorhill** [comment]
https://github.com/gorhill/uBlock/wiki/The-network-request-logger

**Comment by sander85** [comment]
Ok, thanks for the link. But it still doesn't explain requests with white background. With \* \* 3p block I'd say those requests should be blocked but that's not the case always.

**Comment by gorhill** [comment]
> but that's not the case always

`* * 3p block` means network requests are blocked because they are 3rd-party to the page. If you think something is wrong, provide a case please.

**Comment by sander85** [comment]
Don't have any good example currently, but will keep my eyes open about it. But is it possible to change the color for noop requests as well? Yellow/blue maybe? If not then you can close this bug and I'll reopen it if I can reproduce those weird passing requests again.

**Comment by gorhill** [comment]
> is it possible to change the color for noop requests as well

Duplicate of #472.

Even though I am sure many things appear trivial to deal with from the outside, not that trivial to "fix", as a `noop` result may really end up as a `block` or `allow` from a static filter -- so this means it would be reported in an arbitrary manner. I am currently completely unconvinced of the need for this, unlikely to be ever implemented.
